src/starcounter-debug-aid.html

```diff
@@ -173,6 +82,12 @@
                 //this.$.errorDialog.reset();
             },
             openDialog: function () {
+                if (!Polymer.isInstance(this.$.ui)) {
+                    var fileName = "starcounter-debug-aid.html";
+                    var link = document.querySelector('link[href$="' + fileName + '"]');
+                    var baseUrl = link.href.replace(fileName, '');
+                    Polymer.Base.importHref(baseUrl + "starcounter-debug-aid-ui.html");
```

Comment:
Thanks for the suggestion. Lazy import was mentioned twice in #42 so I suppose @diliger considered it but dismissed it for some reason. 

From what I see now, the `group` attribute would actually save us from using a middle element `starcounter-debug-aid-ui.html`. But, at the cost of importing `lazy-imports.html` ;)

On the other hand it looks that `this. importHref` does not have the base URL problem of `Polymer.Base.importHref`.